### PR TITLE
fix caption for hasOne fields

### PR DESCRIPTION
**Pull request by DarkSide666**

### CODE REVIEW

File: src/Reference_One.php [hunk 1]
```diff
@@ -61,6 +61,15 @@ class Reference_One extends Reference
     public $read_only = false;
 
     /**
+     * Defines a label to go along with this field. Use getCaption() which
+     * will always return meaningfull label (even if caption is null). Set
+     * this property to any string.
+     *
+     * @var string
+     */
+    public $caption = null;
```
Comment (romaninsh) [inline src/Reference_One.php]: Are we not extending field?

File: tests/FieldTest.php [hunk 2]
```diff
@@ -107,17 +107,17 @@ public function testMandatory3()
...
-        $m->addField('foo2', ['caption'=>'My Foo']);
-        $this->assertEquals('My Foo', $m->getElement('foo2')->getCaption());
+        $f = $m->addField('foo2', ['caption'=>'My Foo']);
+        $this->assertEquals('My Foo', $f->getCaption());
 
-        $m->addField('foo3', ['ui'=>['caption'=>'My Foo']]);
-        $this->assertEquals('My Foo', $m->getElement('foo2')->getCaption());
+        $f = $m->addField('foo3', ['ui'=>['caption'=>'My Foo']]);
+        $this->assertEquals('My Foo', $f->getCaption());
     }
```
Comment (romaninsh) [inline tests/FieldTest.php]: I think tests were OK Before. But is there test for relations?

File: src/Reference_One.php (hunk 1 above)
Comment (DarkSide666) [inline src/Reference_One.php]: No. We extend Reference which is separate class from Field. That's why we have to redefine these properties here.

File: tests/FieldTest.php (hunk 2 above)
Comment (DarkSide666) [inline tests/FieldTest.php]: Tests were OK. I just made them a bit shorter :)

There is no test for caption in relations. Will do in a short while.

**Pull request merged**